DwayneP.txt

```diff
@@ -1,12 +1,18 @@
-<P>cHello, my name is Dwayne. I am a new student here at Lambdaschool.</p>
+Waddup, I'm D. Yea, I thought about changing my life for the better with some schooling, but I'm content.
 
```

Comment:
although this is a text file don't forget to replace the tags in your new line. for example line 1 was wrapped in <p>tags,
so remember to do the same with yours. For example <p>Waddup, I'm D. Yea, I thought about changing my life for the better with some schooling, but I'm content.</P>  you will want to start getting used to that.